src/luxonis_ml/data/utils/constants.py

```diff
@@ -0,0 +1,17 @@
+from luxonis_ml.enums import LabelType
+from .enums import DataLabelType
+from typing import List, Dict, Union
+
+LDF_VERSION = "1.0.0"
+
+ANNOTATIONS_SCHEMA = {
+    "file": str,  # path to file on local disk or object storage
+    "class": str,  # any string specifying what the annotation is
```

Comment:
Shouldn't "class" and "type" be enums?

Reply:
At first I had `type` as `Union[str, LabelType]`. But I thought it's easier from the user perspective to not import the enums and just use their string values instead. But if we want to force enum use or provide either option it's fine.

For `class` it should definitely be a string since it's user defined (e.g. person, toothbrush, cat, dog). It's the literal class of the annotation.